modules/test/playwright/tests/object-web/objectFolder.spec.ts

```diff
@@ -12,43 +12,123 @@ import {getRandomInt} from '../../utils/getRandomInt';
 
 export const test = mergeTests(apiHelpersTest, loginTest(), objectPagesTest);
 
-test('created object folders are on the left side bar', async ({
-	apiHelpers,
-	viewObjectDefinitionsPage,
-}) => {
-	await viewObjectDefinitionsPage.goto();
+test.describe('Manage object definitions through Model Builder', () => {
+	test('navigate between object folders by Model Builder', async ({
```

Comment:
the `navigate between object folders by Model Builder` is a little bit off to me, i think it would be better to be `navigate between object folders on model builder`. you can see a little explanation why from gemini here:

> The most appropriate choice between "navigate by" and "navigate on" depends on what you're navigating:
> 
> Navigate by: Use this when referring to the method or tool used for navigation. This could be a map, compass, GPS, stars, etc.
> 
> Examples: "The captain navigated by the stars across the open ocean."
> "We navigated through the city by following the bus route on the map."
> Navigate on: Use this when referring to the physical medium you're traveling through.
> 
> Examples: "The plane navigated on autopilot over the Pacific Ocean."
> "The hiker navigated on a narrow, winding trail through the forest."
> Here's a quick trick to remember:
> 
> Navigate by = navigating with something (tool/method)
> Navigate on = navigating over/through something (physical medium)

also i think that the `Model Builder` can be `model builder` as a matter of formatting convention.

Reply:
Hey Rebeca, I agree with the change from by to on 🍻

I chose to use Model Builder starting with capital letters because it is a proper name. But I don't see any problems with using lowercase letters here.

Reply:
`on` is better than `by`, but I would use `inside` :sweat_smile:

Reply:
as in `navigate between object folders inside Model Builder`